data/campaigns/atl02.wmf/scripting/tribes/init.lua

```diff
@@ -0,0 +1,64 @@
+include "tribes/scripting/help/time_strings.lua"
+
+push_textdomain("scenario_atl02.wmf")
+
+local result = {
+   atlanteans = {
+      buildings = {
+         {
+            name = "atlanteans_trading_post",
+            helptexts = {
+               -- TRANSLATORS: Purpose helptext for an atlantean market site: Trading post
+               purpose = pgettext("atlanteans_building", "Enables the share of goods with other players.")
+            }
+         },
+         {
+            name = "atlanteans_scouts_house1",
+            helptexts = {
+               -- TRANSLATORS: Purpose helptext for an atlantean production site: Scout's House
+               purpose = pgettext("atlanteans_building", "Explores unknown territory.")
```

Comment:
Well actually it's the scout who explores unknown territory, not the house he lives in :)

Reply:
it was a complete copy of the original scouts house helptext. However I updated the scenrio version to the actual tribe version. Only purpose is to make the building upgradeable.